tests.toml

```diff
@@ -37,4 +37,4 @@ test_format = 1.0
     # Commits to test upgrade from
     # -------------------------------
 
-    test_upgrade_from.ac288af0.name = "Upgrade from 3.4.0~ynh2"
+    test_upgrade_from.5d1c5af.name = "Upgrade from 3.4.0~ynh2"
```

Comment:
There is a trick in the `preupgrade` script (line 22) where I reuse this commit. The purpose is to test that enabling celery works correctly and the upgrade succeeds with it.

I have amended your commit to update there as well :)